### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by josephmturner**
Add setup.js script to for initial keypair creation.

### CODE REVIEW

File: pull.js [hunk 1]
```diff
@@ -1,19 +1,25 @@
...
+  url: { type: 'string', short: 'u' },
+  location: { type: 'string', short: 'l' },
+  storage: { type: 'string', short: 's'}
+}
+
+const { url, location, storage } = parseArgs({ options }).values
 
-if (!url.startsWith('hyper://')) {
-  console.log('Please specify a `hyper://` URL to load')
+if (url === undefined || !url.startsWith('hyper://')) {
```
Comment (RangerMauve) [inline pull.js]:
```suggestion
if (!url || !url.startsWith('hyper://')) {
```

File: setup.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+import { create } from 'hyper-sdk'
+import { parseArgs } from '@pkgjs/parseargs'
```
Comment (RangerMauve) [inline setup.js]:
```suggestion
import { parseArgs } from 'node:util'
```

File: setup.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  storage: { type: 'string', short: 's'}
+}
+
+const { storage } = parseArgs({ options }).values
+
+const sdk = await create({
+  storage: storage // TODO: Where should the default location for storage be?
+})
+
+console.log(sdk.publicKey.toString('hex'))
```
Comment (RangerMauve) [inline setup.js:15]: What is the purpose of this utility?

**Review by RangerMauve — CHANGES_REQUESTED** [review]

File: setup.js (hunk 3 above)
Comment (josephmturner) [reply setup.js:15]: This utility was for getting the public key for the namespace without pushing or pulling.

**Comment by josephmturner** [comment]
I think this utility isn't relevant to purpose of this example repo.

**Pull request closed without merging**